sdk/cosmos/azure-cosmos/src/samples/java/com/azure/cosmos/ReadmeSamples.java

```diff
@@ -407,12 +409,237 @@ public void databaseCreateContainerPartitionKeySample() {
         // END: com.azure.cosmos.CosmosDatabase.createContainerPartitionKey
     }
 
+    public void cosmosStoredProcedureReadSample1() {
+        String id = "ID";
+        CosmosAsyncStoredProcedure cosmosAsyncStoredProcedure =
+            new CosmosAsyncStoredProcedure(id, cosmosAsyncContainer);
+        // BEGIN: com.azure.cosmos.CosmosStoredProcedure.read_no_params
+        CosmosStoredProcedure procedure = new CosmosStoredProcedure(id, cosmosContainer, cosmosAsyncStoredProcedure);
```

Comment:
why are we mixing sync and async here?

Reply:
Refer to this test file to see how to do it for async and sync - https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/rx/StoredProcedureCrudTest.java 

https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/CosmosSyncStoredProcTest.java